Modules/Office365DSC/Modules/Office365DSCUtil.psm1

```diff
@@ -909,7 +922,7 @@ function Get-TeamByGroupID
     return $true
 }
 
-function Open-SecurityAndComplianceCenterConnection
+function Global:Connect-ExchangeOnline
```

Comment:
Didn't we change this to use the psd1 definition instead?

Reply:
Yes it was added to CmdletsToExport to fix the issue where it was failing the first time it was called.  I don't think I tested it without the explicit Global scope after making that change.  

It might work without declaring Global, since I think CmdletsToExport effectively puts the function into Global scope implicitly.  I'll do some testing.

Reply:
Looks like this indeed continues to work as expected without declaring Global at the function now that it is listed in CmdletsToExport.  Change pushed.